netstandard/ref/System.Reflection.Metadata.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Metadata
+{
+    public static partial class AssemblyExtensions
+    {
+        [System.CLSCompliantAttribute(false)]
+        public unsafe static bool TryGetRawMetadata(this System.Reflection.Assembly assembly, out byte* blob, out int length) { blob = default(byte*); length = default(int); throw null; }
```

Comment:
This API lives in System.Runtime.Loader contract (https://github.com/dotnet/corefx/blob/911dcaa6b49a8bcc1f39270b8c4f233bd21f7f8d/src/System.Runtime.Loader/ref/System.Runtime.Loader.cs#L14), which is the same library as AssemblyLoadContext which I don't think we currently plan to include in the standard. WE should ensure there isn't a particular concern with adding just part of that contract to the standard.

Reply:
Never mind; I thought I excluded the contract but apparently I only excluded the `AssemblyLoadContext` type, which would explain why this type remained in the diff. In that case, let's exclude it for now because it seems "fringy".

@jaredpar, I assume you're OK with that?

Reply:
Yes I'm okay with that.